AK/ReverseIterator.h

```diff
@@ -87,4 +87,18 @@ class SimpleReverseIterator {
     int m_index;
 };
 
+template<typename Container>
+struct reverse_wrapper {
```

Comment:
Please use `TitleCase` for class and struct names, per our coding style.

Reply:
Sorry, forgot it for a while, I have also a timeout on CI, do I need to increase some timeout for testing?